packages/@lightningjs/ui-components/src/components/Slider/SliderLarge.d.ts

```diff
@@ -16,6 +16,12 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import lng from '@lightningjs/core';
 import Slider from '.';
```

Comment:
can we update this import line to `./Slider` so we're not importing from the index file?

Reply:
yep, I was little confused but yeah ./Slider would explicit. Updated, Thanks!